src/spikeinterface/exporters/to_phy.py

```diff
@@ -158,8 +158,9 @@ def export_to_phy(
 
     # export spike_times/spike_templates/spike_clusters
     # here spike_labels is a remapping to unit_index
-    all_spikes = sorting.get_all_spike_trains(outputs="unit_index")
-    spike_times, spike_labels = all_spikes[0]
+    all_spikes_seg0 = sorting.to_spike_vector(concatenated=False)[0]
```

Comment:
Note that Phy only works for 1 segment

Reply:
I know this make the code more explicit no ?